README.md

```diff
@@ -76,11 +76,12 @@ digital-repository/$ cap <stage> deploy
 ```
 
 ##### Digital Hub Export
-* Export repo to JSON formatted for InvenioRDM
-```
-./bin/rake repo_export
-```
-* Same as above, but with extra debugging information
+* To export Digital Hub data to JSON format use the below commands from the root project directory. It's critical to use
```

Comment:
It turns out that the way I ran the command was not the issue with the previous export failing, but I think this is worth putting down here just so we avoid having to relearn this in the future.